pkg/kubelet/winstats/winstats.go

```diff
@@ -103,9 +105,24 @@ func (c *StatsClient) WinContainerInfos() (map[string]cadvisorapiv2.ContainerInf
 }
 
 // WinMachineInfo returns a cadvisorapi.MachineInfo with details about the
-// node machine. Analogous to cadvisor MachineInfo method.
+// node machine. Run the powershell command to get the SystemUUID for Windows node
+// in here if it isn't provided by cadvisor.
 func (c *StatsClient) WinMachineInfo() (*cadvisorapi.MachineInfo, error) {
-	return c.client.getMachineInfo()
+	infos, err := c.client.getMachineInfo()
```

Comment:
Should the code be added `getMachineInfo` instead?

Reply:
+1 to adding it to getMachineInfo()

Reply:
Done.